tools/sgcollect.py

```diff
@@ -279,6 +279,25 @@ def make_http_client_pprof_tasks(
     return pprof_tasks
 
 
+def make_http_client_stack_trace_task(
+    sg_url: str, auth_headers: dict[str, str]
+) -> PythonTask:
+    """
+    This task uses the python http client to collect the raw stack trace data
+    """
+    stack_trace_url = "{0}/_debug/pprof/goroutine?debug=2".format(sg_url)
+
+    stack_trace_task = make_curl_task(
+        name="Collect stack trace via http client",
+        auth_headers=auth_headers,
+        url=stack_trace_url,
+        log_file="sg_stack_trace.log",
```

Comment:
```suggestion
        log_file="goroutines.log",
```

I don't care what we call this but sg_stack_trace.log will mean that this output overwrites the output from the signal handler, so it needs to be a different name.

Basically this is just a plain text representation of pprof_goroutines.pb.gz so I think that would be a reasonable name.

Reply:
I will alter the name for all the files to make it goroutine specific but this doesn't overwrite the signal files, the signal files will have timestamp in the file name, sgcollect http call will not.

Reply:
Oh, that's fine then.